docs/Installation-Windows.md

```diff
@@ -0,0 +1,168 @@
+# Installing ML-Agents for Windows
+In order to get ML-Agents working with Windows, you will need to have Windows 10 installed.  While it is possible for ML-Agents to work on other versions of Windows, we have only tested with a local installation of Windows 10 (not using VM like Bootcamp or Parallels).
```

Comment:
Page-wide feedback:
- Add an empty line after headers 
- You use double spaces to separate sentences (?)

Line-specific feedback:
- "not using VM" --> "not using a VM"

Reply:
Just wondering why we are telling people to use GPU for training specially for Windows users?

Reply:
Perhaps we can have two sub-sections, one for CPU usage (greatly simplified) and the other for GPU usage? It would reduce the number of issues windows-using hobbyists run into by a lot.